packages/hash/api/src/graphql/typeDefs/knowledge/entity.typedef.ts

```diff
@@ -137,11 +129,24 @@ export const persistedEntityTypedef = gql`
     linkedEntities: [PersistedLinkedEntityDefinition!]
   }
 
+  # TODO: rename these and remove "persisted" - https://app.asana.com/0/0/1203157172269854/f
   extend type Query {
     """
-    Get an entity.
+    Get a subgraph rooted at all entities at their latest version.
+    """
+    getAllLatestPersistedEntities(
+      dataTypeResolveDepth: Int!
+      propertyTypeResolveDepth: Int!
+      linkTypeResolveDepth: Int!
+      entityTypeResolveDepth: Int!
+      linkTargetEntityResolveDepth: Int!
+      linkResolveDepth: Int!
+    ): Subgraph!
+
+    """
+    Get a subgraph rooted at an entity resolved by its id.
     """
-    persistedEntity(
+    getPersistedEntity(
```

Comment:
Was there a reason why you weren't explicit about this method now returning a sub-graph?

```suggestion
    persistedEntitySubgraph(
```

Reply:
This is probably worth a discussion, I'd also prefer something explicit. It's a bit difficult and we don't want to be too verbose, the suggestion for example isn't really true. I'd read that like a subgraph _of_ PersistedEntities, but it could have a graph of entities, links, and all the types.